instance/models/openedx_appserver.py

```diff
@@ -87,9 +88,12 @@ class Meta:
 
     # Misc settings:
     use_ephemeral_databases = models.BooleanField()
-    github_admin_organization_name = models.CharField(
-        max_length=200, blank=True, default=settings.DEFAULT_ADMIN_ORGANIZATION,
-        help_text="GitHub organization whose users will be given SSH access to this instance's VMs",
+    github_admin_users = JSONField(
```

Comment:
I considered that approach as well, but in the end decided to go with what I have now.

The help text should make clear that you can have either organizations or users in the list.  We currently don't have any UI for this, and if we add one, we could explain this as well.

Github itself also mixes the two concepts:  You can navigate to either `https://github.com/<user>` or `https://github.com/<org>`, and you can query both types of entities using the `/users` endpoint of the API.  On Github, they are just two different kinds of users, so I treated them the same way on our end.

I don't think the problem with the reverse migration you mention is relevant.  The reverse migration is only provided to make it easier to switch between branches during development.  It will never be run in a production environment.

That said, I don't have a strong preference either way, and since you seem to prefer the other approach, I can just as well go with it.  It actually makes the code a bit simpler, and reduces the number of queries to the Github API.